pkg/cosign/tlog.go

```diff
@@ -63,6 +64,8 @@ const (
 	// TODO(vaikas): Implement storing state like Rekor does so that if tree
 	// state ever changes, it will make lots of noise.
 	addRekorPublicKeyFromRekor = "SIGSTORE_TRUST_REKOR_API_PUBLIC_KEY"
+	// define constants for the proposed entry dynamic spec mapping
+	intotoProposedEntryKey = "intoto"
```

Comment:
can we use intoto.KIND? https://github.com/sigstore/rekor/blob/547eb3cef9690bc97e73e35eb3bf1f40b18a3504/pkg/types/intoto/intoto.go#L28

Reply:
done
f019a63e82219636db34d7d41e909f2ca3008197